modules/installation-configuration-parameters.adoc

```diff
@@ -314,6 +318,10 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
 |The AWS region that the installation program creates compute resources in.
 |Any valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region], such as `us-east-1`.
 
+|`controlPlane.platform.aws.amiID`
+|The AWS AMI used to boot control plane machines for the cluster.
+|Any published or custom {op-system} AMI that belongs to the set AWS region.
+
```

Comment:
I noticed that `compute.platform.aws.amiID` , `controlPlane.platform.aws.amiID` and `platform.aws.amiID` are all under `Optional AWS parameters` table, but either `compute.platform.aws.amiID`, `controlPlane.platform.aws.amiID` or `platform.aws.amiID` is required for usgov region.